elementary/monitor/dbt_project/macros/alerts/get_model_alerts.sql

```diff
@@ -27,37 +27,43 @@
                 union all
                 select unique_id, meta from {{ snapshots_relation }}
             {% endif %}
+        ),
+
+        extended_alerts as (
+            select
+                alerts_in_time_limit.alert_id,
+                alerts_in_time_limit.unique_id,
+                alerts_in_time_limit.detected_at,
+                alerts_in_time_limit.database_name,
+                alerts_in_time_limit.materialization,
+                alerts_in_time_limit.path,
+                alerts_in_time_limit.original_path,
+                alerts_in_time_limit.schema_name,
+                alerts_in_time_limit.message,
+                alerts_in_time_limit.owners,
+                alerts_in_time_limit.tags,
+                alerts_in_time_limit.alias,
+                alerts_in_time_limit.status,
+                alerts_in_time_limit.full_refresh,
+                {# backwards compatibility #}
+                case
+                    when alerts_in_time_limit.suppression_status is NULL and alerts_in_time_limit.alert_sent = TRUE then 'sent'
+                    when alerts_in_time_limit.suppression_status is NULL and alerts_in_time_limit.alert_sent = FALSE then 'pending'
+                    else suppression_status
+                end as suppression_status,
+                case 
+                    when alerts_in_time_limit.sent_at is NULL then '1970-01-01 00:00:00'
```

Comment:
Why not keep it at `NULL`?
I don't see any reason we'd use the UNIX timestamp here.

Reply:
Yeah You are right, this is a leftover from when I developed the feature and didn't handle yet NULL values 🙂